### PR TITLE
Add turn API on variable to docker environment

### LINKED ISSUE #3320: Add turn API on environment variable for docker
### Describe the problem

There is need to turn the API on from docker as discussed here https://community.open-emr.org/t/openemr-flex-docker-updated/12051/40

### Describe the solution you'd like

Add a TURN_API_ON environment variable to docker

### PR DESCRIPTION
**Pull request by prondubuisi**
Fixes #3320

#### Short description of what this resolves:
Add Turn_API_ON environment variable to docker
this environment variable will be used to turn the OpenEMR API on from
within a docker install

### CODE REVIEW

**Comment by bradymiller** [comment]
hi @prondubuisi , looks good. now just gotta get is supported in the docker :)

**Comment by prondubuisi** [comment]
Sure @bradymiller on it.

File: docker-compose.yml [hunk 1]
```diff
@@ -45,6 +45,7 @@ services:
       EASY_DEV_MODE: "yes"
       EASY_DEV_MODE_NEW: "yes"
       DEVELOPER_TOOLS: "yes"
+      TURN_API_ON: "yes"
```
Comment (bradymiller) [inline docker-compose.yml:48]: change this to ACTIVATE_API
(gonna bring this in right after this is fixed to get ready for the new dockers :) )

Comment (prondubuisi) [reply docker-compose.yml]: Done @bradymiller

**Comment by bradymiller** [comment]
hi @prondubuisi , Thanks for the new feature. Hopefully will get dockers out tomorrow that support this. Thanks! -brady

**Pull request merged**